### ISSUE TITLE
FNX3-16619 ⁃ [Bug] Close button in Fenix-launched custom tab is hard to see

### ISSUE THREAD

**Issue by sblatz** [mozilla-mobile/fenix#2836]
The white close button is hard to see against the white toolbar. We just need to pass in a theming option (like other apps do) to handle this.
![close tab](https://user-images.githubusercontent.com/4400286/58348014-3e3ad700-7e14-11e9-87b3-ad66eed5836a.gif)

**Comment by jonalmeida** [comment]
In the a-c feature we tint the buttons depending on the toolbar colour we get. By using the custom tab internally the app is circumventing this value being set. The quickest fix is to include the themed toolbar colour in your intent like below and that should take care of the rest:

```diff
diff --git a/app/src/main/java/org/mozilla/fenix/settings/SupportUtils.kt b/app/src/main/java/org/mozilla/fenix/settings/SupportUtils.kt
index adfd6290..edc1cd29 100644
--- a/app/src/main/java/org/mozilla/fenix/settings/SupportUtils.kt
+++ b/app/src/main/java/org/mozilla/fenix/settings/SupportUtils.kt
@@ -45,6 +45,7 @@ object SupportUtils {
     }

     fun createCustomTabIntent(context: Context, url: String) = Intent(Intent.ACTION_VIEW).apply {
+        putExtra("android.support.customtabs.extra.TOOLBAR_COLOR", ContextCompat.getColor(context, R.color.white_color))
         putExtra("android.support.customtabs.extra.SESSION", true)
         setClassName(context.applicationContext, IntentReceiverActivity::class.java.name)
         data = Uri.parse(url)
```

**Comment by softvision-miralobontiu** [comment]
Verified as fixed on the latest build 1.0.1922 (#11530616), with Nokia 6 (Android 7.1.1), Motorola Moto G6 (Android 8), Samsung Galaxy Tab S3 (Android 8), and Huawei P9 Lite (Android 6.0).
